src/uvw/util.hpp

```diff
@@ -507,21 +513,21 @@ struct Utilities {
 
         if(0 == uv_interface_addresses(&ifaces, &count)) {
             std::for_each(ifaces, ifaces+count, [&interfaces](const auto &iface) {
-                InterfaceAddress interface;
+                InterfaceAddress tInterface;
```

Comment:
They are aggregates, I'd rather exploit this. I'm modifying the commit on the fly, let me know if it works for you (it should).